src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+
+import io.qameta.allure.Step;
+import org.openqa.selenium.*;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.interactions.Actions;
+
+public class HomePage extends AbstractPage {
+
+    private final By signInBtnLocator = By.linkText("Sign In");
```

Comment:
The way you locate element is correct, but not preferable.
Normally, the first option is 'id', the second css, third xpath, then everything else. 
For this element you could use 'header .authorization-link' css locator.
The reason behind it: 
- Apps are very often have multiple languages
- Simpler to maintain. If the test is failing and you want to debug, having id, css or xpath gives you opportunity to check in two clicks if element is in place with dev tools. 
- Fragile. Text could be changed. This point is questionable, there are various opiniions.

Reply:
Thank you for the comment, got it! Replaced locators with linkText where possible, there are only 2 left (colorFilterLocator, priceFilterLocator). It was hard to find other options for them